server/src/repositories/storage.repository.ts

```diff
@@ -172,10 +174,15 @@ export class StorageRepository implements IStorageRepository {
       return emptyGenerator();
     }
 
-    const stream = globStream(this.asGlob(pathsToCrawl), {
+    const globbedPaths = pathsToCrawl.map((path) => this.asGlob(path));
+
+    this.logger.debug(globbedPaths);
+
+    const stream = globStream(globbedPaths, {
       absolute: true,
       caseSensitiveMatch: false,
       onlyFiles: true,
+      unique: true,
```

Comment:
Don't add this. It makes the glob slower and use more memory.

Reply:
Ok, will remove!